File: pset6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if (height > 0 and height < 9):
        break

for i in range(height):
    for j in range(height - 1 - i):
        print(" ", end="")
    for t in range(i + 1):
        print("#", end="")
    for s in range(2):
        print(" ", end="")
    for d in range(i + 1):
        print("#", end="")
    print()

File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)


{
    // taking in correct input
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

// function for printing ..

    for (int i = 0; i < n ; i++)
    {
        // function for printing space

        for (int j = n - i - 1 ; j > 0 ; j--)
        {
            printf(" ");
        }
        // function for printing #

        for (int k = 0 ; k <= i ; k++)
        {
            printf("#");
        }

        // function for printing double space

        printf("  ");

        for (int l = 0 ; l <= i ; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}